go/internal/test_tools/test_tools.go

```diff
@@ -88,3 +88,14 @@ func GetPort() string {
 		return port
 	}
 }
+
+// GetAuth returns the auth string to connect to for the tests.
+// By default it is Anonymous, but can be overridden by setting the DH_AUTH environment variable.
+func GetAuth() string {
+	auth := os.Getenv("DH_AUTH")
+	if auth == "" {
+		return "Anonymous"
```

Comment:
Should this be `Constants.DefaultAuth` (or whatever) rather than the literal string "Anonymous"?

Reply:
I don't think this should be using the constant.  This is set to match the DH instance that is spun up to test against.  We have no guarantee that the instance will be using the default connection mechanism, so I'm not completely comfortable linking them.

Reply:
My opinion is weak on this, so if you think it should be changed, I will.

Reply:
I'm not deeply invested in either choice. I assumed it was an oversight. If not, that's fine.